### PR TITLE
[#15241] Proper system message on being not a member of a group

### LINKED ISSUE #15241: `Add <no value> to send message` on mobile after removing user from group chat from desktop
# Bug Report

## Problem 
After a user is removed from a group chat, created on the desktop, `Add <no value> to send message`  is shown

#### Expected behavior
Can't send a message and see the appropriate system message

#### Actual behavior
IMAGE 2023-03-03 13:43:17

### Reproduction
Mobile and Desktop users are mutual contacts.

1) Desktop: create group chat with Mobile user
2) Desktop: remove Mobile user from Group chat

### Additional Information

- Status version: nightly 03/03/2023
- Operating System: Android, iOS

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by rasom**
fix #15241

status: ready

### CODE REVIEW

**Comment by rasom** [comment]
@churik is that what we need?

Screenshot 2023-03-06 at 18 40 02

File: src/status_im2/contexts/chat/messages/contact_requests/bottom_drawer.cljs [hunk 1]
```diff
@@ -7,14 +7,18 @@
     [status-im2.constants :as constants]))
 
 (defn view
-  [contact-id contact-request-state]
+  [contact-id contact-request-state group-chat]
+  (println :KUKUHA contact-id contact-request-state)
```
Comment (briansztamfater) [inline src/status_im2/contexts/chat/messages/contact_requests/bottom_drawer.cljs]: remove println

Comment (rasom) [reply src/status_im2/contexts/chat/messages/contact_requests/bottom_drawer.cljs]: oops, done

**Comment by churik** [comment]
@rasom yes, let me know when it is ready to test, thank you!

**Comment by rasom** [comment]
@churik it is ready

**Comment by churik** [comment]
ready to be merged after approve, thank you!

**Review by ibrkhalil — APPROVED** [review]

**Pull request merged**